flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -777,15 +910,22 @@ class FlxTilemapExt extends FlxTilemap
 		
 		if (value != null && _specialTiles != null && _specialTiles.length > 0)
 		{
-			for (t in _specialTiles) 
+			for (tile in _specialTiles) 
 			{
-				if (t != null) 
+				if (tile != null) 
 				{
-					t.frames = frames;
+					tile.frames = frames;
 				}
 			}
 		}
 		
 		return value;
 	}
+}
+
+enum FlxSlopeSteepness
+{
+	GENTLE; // 22.5°
```

Comment:
This type of comment doesn't work properly for completion, it has to be a `/** */`-style comment _before_ the actual enum value - see `FlxCameraFollowStyle` for an example.